diff --git a/dist/runtime/parser/utils/html-tags-list.js b/dist/runtime/parser/utils/html-tags-list.js
index f8e905552787d0341d6aaedf5886a662eab28914..a9cf6a4c29214f6e9e63225814ca5d68598666ae 100644
--- a/dist/runtime/parser/utils/html-tags-list.js
+++ b/dist/runtime/parser/utils/html-tags-list.js
@@ -115,5 +115,34 @@ export default [
   "ul",
   "var",
   "video",
-  "wbr"
+  "wbr",
+'annotation',
+'semantics',
+'mtext',
+'mn',
+'mo',
+'mi',
+'mspace',
+'mover',
+'munder',
+'munderover',
+'msup',
+'msub',
+'msubsup',
+'mfrac',
+'mroot',
+'msqrt',
+'mtable',
+'mtr',
+'mtd',
+'mlabeledtr',
+'mrow',
+'menclose',
+'mstyle',
+'mpadded',
+'mphantom',
+'mglyph',
+'svg',
+'line',
+'path'
 ];
