diff --git a/dist/module.mjs b/dist/module.mjs
index 484c3b0a407c0edae3aa2690bc3e56b8489f5a94..cacb3824409e03ef2c78ab24b0643012059e0a47 100644
--- a/dist/module.mjs
+++ b/dist/module.mjs
@@ -590,7 +590,7 @@ const module = defineNuxtModule({
       return output;
     }, {});
     if (Object.keys(sources).length === 0 || !sources.content) {
-      const content = nuxt.options.srcDir + "/content";
+      const content = nuxt.options.rootDir + "/content";
       if (Fs.existsSync(content)) {
         sources.content = {
           driver: "fs",
